cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -751,6 +751,10 @@
      * Structured table metrics
      */
     public static final class StructuredTable {
+      public static final Map<String, String> METRICS_TAGS = ImmutableMap.of(
+        Constants.Metrics.Tag.COMPONENT, Constants.Service.DATASET_MANAGER,
```

Comment:
Rather than using `dataset.service` as the component, can you use something like `system.storage`?

Reply:
Fixed